{
  ServerSocket socket=null;
  try {
    socket=new ServerSocket(0);
  }
 catch (  IOException e) {
    e.printStackTrace();
    Assert.fail("An exception was thrown while preparing the test " + e.getMessage());
  }
  Configuration conf=new Configuration();
  conf.setString(ConfigConstants.BLOB_SERVER_PORT,String.valueOf(socket.getLocalPort()));
  try {
    BlobServer srv=new BlobServer(conf);
  }
  finally {
    socket.close();
  }
}
